### ISSUE TITLE
Annotatins name

### ISSUE THREAD

**Issue by WarGot-by** [alpixel/AlpixelCronBundle#8]
May be rename value annotation to repeat ?

**Comment by mwansinck** [comment]
I agree with renaming value to something else, however, I would suggest interval (to match the PHP format DateInterval) or frequency.

**Comment by benjamin-hubert** [comment]
I agree, this will be on next release. Will think about a deprecated strategy. 
(sorry about the time being away and the long delay .. !)
